swift-sdk/Internal/EmbeddedMessagingSerialization.swift

```diff
@@ -4,6 +4,38 @@
 
 import Foundation
 
+struct AnyDecodable: Decodable {
```

Comment:
Not sure if there should be a generic `AnyDecodable` struct. InApps also supports custom payload but seem to have another approach to it.